README.md

```diff
@@ -1,4 +1,56 @@
-# ActiveModelSerializers
+# ActiveModelSerializers (Deprecated)
+
+Please use [jsonapi-rb](https://github.com/jsonapi-rb) ([docs](http://jsonapi.org))
```

Comment:
This isn't a totally fair recommendation since not everyone wants JSON:API, and jsonapi-rb isn't the most-obvious choice for JSON:API.  If we want to be fair, we should suggest ROAR, jsonapi-resources, etc. 

I'd be happy to add an alternatives section down where we mention dev might not be released, but generally I think this recommendation is too specific for one library.